cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -166,6 +166,20 @@ func copyUbuntuDataAuth(src, dst string) error {
 		// so that users have proper perms, i.e. console-conf added users are
 		// sudoers
 		"system-data/etc/sudoers.d/*",
+		// for network configuration setup by console-conf, etc.
+		// TODO:UC20: we want some way to "try" or "verify" the network
+		//            configuration before installing it onto recover mode,
+		//            because the network configuration could have been what was
+		//            broken so we don't want to break network configuration for
+		//            recover mode as well, but for now this is fine
+		"system-data/etc/netplan/*",
```

Comment:
this is not auth data anymore, we probably want a different helper all together even because at some point this kind of data in recovery might also come from the save partition

Reply:
Let's add a new extra helper "copyNetworkConfig()" - I will do this.